### ISSUE TITLE
Update Esperanto translations

### ISSUE THREAD

**Issue by janvanveldhuizen** [squidfunk/mkdocs-material#6744]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "eo",
  "action.edit": "Redakti ĉi tiun paĝon",
  "action.skip": "Saltu al enhavo",
  "action.view": "Vidi fonton de ĉi tiu paĝo",
  "announce.dismiss": "Ne montru ĉi tion denove",
  "blog.archive": "Arkivo",
  "blog.categories": "Kategorioj",
  "blog.categories.in": "en",
  "blog.continue": "Daŭrigi legadon",
  "blog.draft": "Malneto",
  "blog.index": "Reen al indekso",
  "blog.meta": "Metadatenoj",
  "blog.references": "Rilataj ligiloj",
  "clipboard.copy": "Kopii al tondujo",
  "clipboard.copied": "Kopiado al klipo",
  "consent.accept": "Akcepti",
  "consent.manage": "Administri agordojn",
  "consent.reject": "Rifuzi",
  "footer": "Piedlinio",
  "footer.next": "Sekva",
  "footer.previous": "Antaŭa",
  "header": "Kaplinio",
  "meta.comments": "Komentoj",
  "meta.source": "Fontkodo",
  "nav": "Navigado",
  "readtime.one": "1 min legado",
  "readtime.other": "# min legado",
  "rss.created": "RSS-fluo",
  "rss.updated": "RSS-fluo de ĝisdatigita enhavo",
  "search": "Serĉi",
  "search.placeholder": "Serĉo",
  "search.share": "Kunhavigi",
  "search.reset": "Klara",
  "search.result.initializer": "Komencante serĉon",
  "search.result.placeholder": "Tajpu por komenci serĉadon",
  "search.result.none": "Neniuj kongruaj dokumentoj",
  "search.result.one": "1 kongrua dokumento",
  "search.result.other": "# kongruaj dokumentoj",
  "search.result.more.one": "1 pli sur ĉi tiu paĝo",
  "search.result.more.other": "# pli sur ĉi tiu paĝo",
  "search.result.term.missing": "Mankanta",
  "select.language": "Elekti lingvon",
  "select.version": "Elekti version",
  "source": "Iru al deponejo",
  "source.file.contributors": "Kontribuantoj",
  "source.file.date.created": "Kreita",
  "source.file.date.updated": "Lasta ĝisdatigo",
  "tabs": "Langetoj",
  "toc": "Enhavtabelo",
  "top": "Reen al supro"
}[key] }}{% endmacro %}
```

### Country flag

:flag_eu:

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to the best of my knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by janvanveldhuizen** [comment]
There is an Esperanto flag, but not in the Emojipedia.
Currently the EU flag is linked, which is not quite appropriate for a non-political worldwide language.
This is a link to image files of the proper flag: https://icons8.com/icon/bcwQm25Fwvxw/esperanto-flag
I have no idea if this can be used somehow.

**Comment by squidfunk** [comment]
Thanks for the updated translations! We know that the flag situation is not perfect (e.g. we're using the Spanish flag for Catalan as well), but we just try to find the closest possible flag in order to give a visual aid, so I guess we just stay with the EU flag. We only support flags contained in the Twemoji icon set.

**Comment by squidfunk** [comment]
Added in 1b398c0ad.

**Comment by squidfunk** [comment]
Released as part of [9.5.8](https://squidfunk.github.io/mkdocs-material/changelog/#9.5.8).
